src/Context/filterContext.js

```diff
@@ -0,0 +1,122 @@
+import {
+    createContext,
+    useContext,
+    useReducer, 
+    useState, 
+    useEffect
+} from "react";
+
+import axios from "axios";
+
+const initVal = {
+    categoryFil: [],
+    priceRangeFil: 500,
+    ratingFil: 1,
+    sortByPriceFil: "",
+    searchFil: "",
+    showProduct: [],
+    realProduct: []
```

Comment:
Suggestion: You can give more specific name .

Reply:
Okay will do that.